### PR TITLE
Np 7909/fix filters

### PR DESCRIPTION
**Pull request by sarthak-marwaha**

### CODE REVIEW

File: src/foam/parse/QueryParser.js [hunk 1]
```diff
@@ -106,8 +106,7 @@ foam.CLASS({
             sym('dot'),
             sym('equals'),
             sym('before'),
-            sym('after'),
-            sym('id')
```
Comment (smirnovaka) [inline src/foam/parse/QueryParser.js:110]: @kgrgreer why was it here

Comment (kgrgreer) [reply src/foam/parse/QueryParser.js:110]: It is for searching by primary key. But it looks like it only works for integer ids as it is currently implemented.

**Comment by kgrgreer** [comment]
While testing with @sarthak-marwaha, this was working fine.

**Comment by kgrgreer** [comment]
Seems to be working fine now. Waiting for @adamfox-nanopay to verify.

**Comment by kgrgreer** [comment]
it is not working in 4.24 but is working in development

**Pull request closed without merging**